pyproject.toml

```diff
@@ -26,7 +26,7 @@ requires-python = ">=3.10"
 readme = "README.md"
 keywords = ["neuroscience", "biophysics", "simulator", "jax"]
 dependencies = [
-    "jax",
+    "jax <= 0.4.31",
```

Comment:
Let's link this issue in a comment here, in case ppl wonder / wanna use newer jax versions.